tests/integration/main.go

```diff
@@ -24,45 +25,47 @@ var localSdkPath = flag.String("local-sdk-path", "",
 func main() {
 	flag.Parse()
 
-	// wg waits for all runners to complete
-	var wg sync.WaitGroup
+	if happyPathOnly != nil && *happyPathOnly {
+		tr := DefaultTestRun()
+		tr.SetLocalSDKPath(*localSdkPath)
+		tr.ValidateStringLiterals()
+		tr.startDocker()
+		tr.ExecuteSteps(happyPathSteps)
+		tr.teardownDocker()
+		return
+	}
 
 	start := time.Now()
 	tr := DefaultTestRun()
 	tr.SetLocalSDKPath(*localSdkPath)
 	tr.ValidateStringLiterals()
 	tr.startDocker()
+	tr.ExecuteSteps(happyPathSteps)
```

Comment:
Could we add a comment or TODO that currently everything runs sequentially, but we'll build out concurrency eventually